tests/CMakeLists.txt

```diff
@@ -120,7 +141,9 @@ if (FFTW_FOUND)
 endif ()
```

Comment:
When `VEXCL_BACKEND=All`, target names for the fft test will be different. Using `vexcl_add_executables` should help here as well.

Reply:
Took a shot at fixing that when fixing the issue below.